Gemfile

```diff
@@ -18,18 +18,18 @@ gem "pg_search"
 gem "kaminari"
 
 # Use Puma as the app server
-gem "puma", "~> 3.11"
+gem "puma"
 # Use SCSS for stylesheets
-gem "sass-rails", "~> 5.0"
+gem "sass-rails"
 # Use Uglifier as compressor for JavaScript assets
-gem "uglifier", ">= 1.3.0"
+gem "uglifier"
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'mini_racer', platforms: :ruby
 
 # Heroku ruby buildpack installs yarn only when webpacker gem is detected...
 gem "webpacker", require: false
 
-gem "font-awesome-rails", ">= 4.7", github: "bokmann/font-awesome-rails", branch: "master"
+gem "font-awesome-rails"
```

Comment:
I guess at the time there was some patch not yet published on rubygems, but now it's available so we can source it from rubygems the regular way